packages/volto/src/components/manage/Widgets/FormFieldWrapper.jsx

```diff
@@ -112,16 +122,14 @@ class FormFieldWrapper extends Component {
           description ? 'help' : '',
           className,
           `field-wrapper-${id}`,
-          multilingual_options?.language_independent
-            ? 'language-independent-field'
-            : null,
+          languageIndependent ? 'language-independent-field' : null,
         )}
       >
         <Grid>
           <Grid.Row stretched>
             {columns === 2 && (
               <Grid.Column width="4">
-                <div className="wrapper">
+                <div className="wrapper title-icon-container">
```

Comment:
Seeing all the snapshots that were updated makes me wonder:
1. Do we need the `title-icon-container` class even for the fields that don't have an icon?
2. Should we just apply the flex styles to all field wrappers and not add the new class?
/cc @sneridagh

Reply:
I agree with David, although I can see that write a class enough specific to match only that wrapper can be difficult. @iRohitSingh can you look into that?